service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DeserializeAndAcceptEmptyStringsAsNullObjectTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.io.File;
+import java.io.IOException;
+
+@ActiveProfiles("integration-test")
```

Comment:
Also don't think this should be in integration tests, more of a unit test sort of thing

Reply:
@akomisarek had thought integration, should I move to unit?

Reply:
I was thinking integration because it relies on our app configuration, the problem with unit test is you have got different (specifically) created ObjectMapper for the unit test. Where as here partially we test our configuration and if spring pulls the setting properly - so I think it is sort of int test. (We probably wouldn't need WebMvcTest, but I think it's simpler just to be consistent), we need spring boot test with our config which is www config, so I don't mind massively. Thoughts?

Reply:
Yeah makes sense forgot about that until I tried it out